sycl/test/basic_tests/range_zero_size.cpp

```diff
@@ -0,0 +1,18 @@
+// REQUIRES:cuda || hip_amd || opencl || host
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+//==--------------- range_zero_size.cpp - SYCL range test ------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <sycl/sycl.hpp>
+using namespace sycl;
+
+int main() {
+  queue q;
+  q.submit(
+      [&](handler &cgh) { cgh.parallel_for(range<1>(0), [=](id<1> i) {}); });
+}
```

Comment:
Device-dependent testing should be in https://github.com/intel/llvm-test-suite rather than in the in-tree LIT tests. Could you please move this test to there?

Reply:
actually, if you agree, the device dependent part of the test to be removed. The same behaviour should be seen for all backends.